nevergrad/benchmark/xpbase.py

```diff
@@ -129,6 +135,9 @@ def __init__(self, function: fbase.ExperimentFunction,
         self.result = {"loss": np.nan, "elapsed_budget": np.nan, "elapsed_time": np.nan, "error": ""}
         self.recommendation: Optional[p.Parameter] = None
         self._optimizer: Optional[obase.Optimizer] = None  # to be able to restore stopped/checkpointed optimizer
+        # make sure the random_state of the base function is created, so that spawning copy does not
+        # trigger a seed for the base function, but only for the copied function
+        self.function.parametrization.random_state  # pylint: disable=pointless-statement
```

Comment:
Here was the reproducibility issue, that was a problem of my code. There was an extra call of the default random generator to seed this parametrization after the beginning of the first experiment only, while this parametrization is not used in the experiments.